qt/setup.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Hasnt utf8 been the default for a long time now?

Reply:
Looks like they are:
1. https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3
1. https://discuss.python.org/t/pep-597-use-utf-8-for-default-text-file-encoding/1819